next.clayui.com/src/components/clay/Icon.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayIcon, {ClayIconSpriteContext} from '@clayui/icon';
+import Editor from './Editor';
+import React from 'react';
+
+const spritemap = '/images/icons/icons.svg';
+
+export const Icon = () => {
+	const scope = {ClayIcon, spritemap};
+	const code = `const Component = () => {
```

Comment:
Can we indent this template to be on the same line? Otherwise it looks a bit weird.

Reply:
When Identing the template in this way,
<img width="442" alt="Screen Shot 2019-06-17 at 19 11 01" src="https://user-images.githubusercontent.com/7663513/59640212-bb1b5100-9133-11e9-811f-7233312964bf.png">

the output in the example will look a bit weird, see:
<img width="632" alt="Screen Shot 2019-06-17 at 19 11 09" src="https://user-images.githubusercontent.com/7663513/59640269-e9009580-9133-11e9-963f-21af8eea0030.png">

Reply:
A possible workaround for this is:
<img width="398" alt="Screen Shot 2019-06-17 at 19 13 38" src="https://user-images.githubusercontent.com/7663513/59640354-2f55f480-9134-11e9-83dc-f5c42a7020e4.png">

What do you think? @matuzalemsteles @bryceosterhaus

Reply:
hmm, still looks weird to me.

What if we did something like this and move it outside of the function.
```tsx
const iconWithContextCode = `const Component = () => {
	return (
		<ClayIconSpriteContext.Provider value={spritemap}>
			<ClayIcon symbol="add-cell" />
		</ClayIconSpriteContext.Provider>
	);
}

render(<Component />)`;

export const IconWithContext = () => {
	const scope = {ClayIcon, ClayIconSpriteContext, spritemap};

	return <Editor code={iconWithContextCode} scope={scope} />;
};
```

Reply:
I think this might look better!

Reply:
LGTM